aarch64/src/devcons.rs

```diff
@@ -6,10 +6,8 @@ use core::ptr;
 use core::ptr::{read_volatile, write_volatile};
 use port::devcons::{Console, Uart};
 use port::fdt::{DeviceTree, RegBlock};
-use spin::Once;
 
 static mut UART: MaybeUninit<Pl011Uart> = MaybeUninit::uninit();
```

Comment:
I think you can move this down into the closure that you pass into devcons, but otherwise, this looks ok.